interface-definitions/include/accel-ppp/radius-additions.xml.i

```diff
@@ -57,6 +57,24 @@
           </properties>
           <defaultValue>0</defaultValue>
         </leafNode>
+        <leafNode name="priority">
```

Comment:
Please use a common `#include` for the priority node.

Move https://github.com/vyos/vyos-1x/blob/current/interface-definitions/system_login.xml.in#L205C1-L217C30 to `include/radius-priority.xml.i` and include the snipped in both places. Only set defaultValue 255 for system-login

Reply:
done

Reply:
```
vyos@vyos# set vpn l2tp remote-access authentication radius server 192.168.0.1 [tab]
Possible completions:
   acct-port            Accounting port (default: 1813)
   backup               Use backup server if other servers are not available
   disable              Disable instance
   disable-accounting   Disable accounting
   fail-time            Mark server unavailable for <n> seconds on failure (default: 0)
   key                  Shared secret key
   port                 Port number used by connection (default: 1812)
   priority             Server priority
```

```
vyos@vyos# set system login radius server 192.168.0.1 [tab]
Possible completions:
   disable              Disable instance
   key                  Shared secret key
   port                 Port number used by connection (default: 1812)
   priority             Server priority (default: 255)
   timeout              Session timeout (default: 2)
```